gitops/pipelines/pipelines-crds/overlays/patched/kustomization.yaml

```diff
@@ -1,4 +1,5 @@
 resources:
+- pods.yaml
```

Comment:
Do we need this and the pod.yaml file itself? 
I think with the new syncer mode, we just need to specify pods under resources to sync in the syncer.yaml file. Please verify.

Reply:
See previous comment.